plugins/flutter_aepedgeconsent/pubspec.yaml

```diff
@@ -13,7 +13,6 @@ dependencies:
   flutter:
     sdk: flutter
   flutter_aepcore: ">= 1.0.0 <2.0.0"
-  #flutter_aepedgeidentity: ">= 1.0.0 <2.0.0"
```

Comment:
Does this not depend on aepedgeidentity?

Reply:
Consent doesn't.  Only Edge Networks plugin depends on aepedgeidentity.